tutorial.md

```diff
@@ -20,16 +27,20 @@ these [step by step instructions](kind-setup.md) to install on your local
 machine.
 
 ## Boson Tooling
+
 The primary interface for Boson project is the `faas` CLI.
 [Download][faas-download] the most recent version and install it some place
```

Comment:
Just curious why don't you use this syntax?

`[Download](https://github.com/boson-project/faas/releases)`

Reply:
The lines were really long with the URLs inline. I was just trying to keep it a little neater in the text.